playbook.yml

```diff
@@ -80,6 +80,10 @@ content:
     - url: https://github.com/OpenZeppelin/nile
       branches: release-v*
       start_path: docs
+    
+    - url: https://github.com/OpenZeppelin/polkadot-runtime-template
```

Comment:
It looks like this redirects to https://github.com/OpenZeppelin/polkadot-generic-runtime-template.  Should this url be changed to https://github.com/OpenZeppelin/polkadot-generic-runtime-template ?

Reply:
Good catch!
Will fix in a moment.